tests/formulas/rsa/rsa_condition_nationalite.yaml

```diff
@@ -0,0 +1,7 @@
+- period: 2018-06
+  input:
+    nationalite: [null, null, null, null, FR]
```

Comment:
C'est dommage d'utiliser la valeur non représentative "null" comme input, pourquoi pas des valeurs correspondant aux valeurs de `ressortissant_eee` ? Par exemple [DE, BE, AO, NZ, FR] ?

Reply:
Pourquoi "dommage" ?
Ce que je veux tester, c'est le comportement en fonction de la valeur de `ressortissant_eee`, mais comme par défaut, la nationalité est mise à FR, je voudrais juste remettre ça à zéro pour que ça n'impacte pas le test. Ca te semble être une mauvaise façon de faire ?

Reply:
Mon raisonnement est le même que pour les codes à trois lettres - il vaut mieux que les valeurs en input dans un test fassent partie du domaine "officiel" des valeurs. Ici `null` n'est pas une valeur attendue pour la variable `nationalite` - on n'a pas besoin d'exiger du code qu'il soit capable de traiter cette valeur.